website/docs/guides/bigquery-qs.md

```diff
@@ -110,10 +110,18 @@ Now that you have a repository configured, you can initialize your project and s
     - In the command line bar at the bottom, enter `dbt run` and click **Enter**. You should see a `dbt run succeeded` message.
 
 ## Build your first model
-1. Under **Version Control** on the left, click **Create branch**. You can name it `add-customers-model`. You need to create a new branch since the main branch is set to read-only mode.
-3. Click the **...** next to the `models` directory, then select **Create file**.  
-4. Name the file `customers.sql`, then click **Create**.
-5. Copy the following query into the file and click **Save**.
+
+In the dbt Cloud IDE, you have two options for working with files:
+
+- Direct editing in the protected primary branch &mdash; You can edit, format, or lint files and execute dbt commands directly in your primary git branch. Once you're ready, you can commit those changes to a new branch.
```

Comment:
```suggestion
- Direct editing in the protected primary branch &mdash; You can edit, format, or lint files and execute dbt commands directly in your primary git branch. Since the dbt Cloud IDE prevents commits to the protected branch, you can commit those changes to a new branch.
```